### PR TITLE
redesigned admin cafe, fixes leaks

### PR DESCRIPTION
**Pull request by buffyuwu**
also fixed blob announcement

### CODE REVIEW

File: code/modules/events/viral_outbreak.dm [hunk 1]
```diff
@@ -8,7 +8,7 @@ datum/event/viral_outbreak/setup()
 	severity = rand(2, 4)
 
 datum/event/viral_outbreak/announce()
-	command_announcement.Announce("Confirmed outbreak of level 7 biohazard aboard the [location_name()]. All personnel must contain the outbreak.", "Biohazard Alert", new_sound = GLOB.using_map.level_x_biohazard_sound(7))
+	command_announcement.Announce("Confirmed containment breach of KETER class object. All security personnel must recontain the object.", "Biohazard Alert", new_sound = GLOB.using_map.level_x_biohazard_sound(7))
```
Comment (kachnov) [inline code/modules/events/viral_outbreak.dm:11]: this doesn't make sense, why is a virus KETER

Comment (buffyuwu) [reply code/modules/events/viral_outbreak.dm:11]: its the blob you dip

Comment (kachnov) [reply code/modules/events/viral_outbreak.dm:11]: o.

**Review by kachnov — DISMISSED** [review]

**Review by JakeDaBoss — CHANGES_REQUESTED** [review]
Awaiting inspection.

File: code/game/machinery/doors/door.dm [hunk 2]
```diff
@@ -525,3 +525,26 @@
 
 /obj/machinery/door/morgue
 	icon = 'icons/obj/doors/doormorgue.dmi'
+
+/obj/machinery/cryopod/door/escape/
```
Comment (kachnov) [inline code/game/machinery/doors/door.dm]: remove the trailing slash

Comment (kachnov) [inline code/game/machinery/doors/door.dm]: i'm not sure why you chose to use a cryopod for this

File: code/game/machinery/doors/door.dm [hunk 3]
```diff
@@ -525,3 +525,26 @@
...
+	name = "\improper Desert"
+	time_till_despawn = 10 SECONDS
+	on_store_message = "has been detected departing the facility via the desert."
+	on_enter_occupant_message = "The desert sprawls out over the horizon, and the sun burns down on you from above."
+
+/obj/machinery/cryopod/door/escape/Bumped(var/atom/movable/AM)
+	if(!ishuman(AM))
+		return
+
+	var/mob/living/carbon/human/user = AM
```
Comment (kachnov) [inline code/game/machinery/doors/door.dm]: why are there two mobs

File: code/game/machinery/doors/door.dm [hunk 4]
```diff
@@ -525,3 +525,26 @@
...
+
+/obj/machinery/cryopod/door/escape/
+	name = "\improper Desert"
+	time_till_despawn = 10 SECONDS
+	on_store_message = "has been detected departing the facility via the desert."
+	on_enter_occupant_message = "The desert sprawls out over the horizon, and the sun burns down on you from above."
+
+/obj/machinery/cryopod/door/escape/Bumped(var/atom/movable/AM)
+	if(!ishuman(AM))
+		return
```
Comment (kachnov) [inline code/game/machinery/doors/door.dm]: return ..(AM)

File: code/game/machinery/doors/door.dm [hunk 5]
```diff
@@ -525,3 +525,26 @@
...
+	var/choice = alert("Do you want to depart via the desert? Your character will leave the round.","Departure","Yes","No")
+	if(user && Adjacent(user) && choice == "Yes")
+		user.ghostize()
+
+		job_master.FreeRole(H.mind.assigned_role)
+
+		message_admins("\blue [key_name(AM)] has escaped via the desert")
+		log_admin("\blue [key_name(AM)] has escaped via the desert")
+
+		del(AM)
```
Comment (kachnov) [inline code/game/machinery/doors/door.dm]: use qdel

File: code/game/machinery/doors/door.dm [hunk 6]
```diff
@@ -525,3 +525,26 @@
...
+	if(!ishuman(AM))
+		return
+
+	var/mob/living/carbon/human/user = AM
+	var/mob/living/carbon/human/H = AM
+	var/choice = alert("Do you want to depart via the desert? Your character will leave the round.","Departure","Yes","No")
+	if(user && Adjacent(user) && choice == "Yes")
+		user.ghostize()
+
+		job_master.FreeRole(H.mind.assigned_role)
```
Comment (kachnov) [inline code/game/machinery/doors/door.dm]: do this before you ghostize them

**Review by kachnov — CHANGES_REQUESTED** [review]

**Review by kachnov — APPROVED** [review]

**Comment by JakeDaBoss** [comment]
Please present the changes to me in a DM, and the affected map file to make integration into the update smoother.

**Pull request closed without merging**